src/components/button/examples/button-reduce-presence.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * Reduce Presence
+ *
+ * This example is identical to the one above, except that here, the
+ * `reducePresence` property has been set to `true`. This will hide the button
+ * when it is disabled. However, it will also make sure that the button remains
+ * visible while the loading animation is ongoing. When the animation is done
+ * and the checkmark has been shown, the button will hide.
+ *
+ * ###### Usage of `reducePresence` property
```

Comment:
I think everything from this heading down is a bit too long-form and philosophical to be a good fit here in the docs for the example. I think it might fit better under the design guidelines, and then we could have a more concise one-paragraph description here, with a link to the longer text.

Reply:
Hmm... 🤔  Good idea. What do we call that section in Design Guidelines? 
Perhaps: "**Reducing UI clutter**"

Can you make this sections? Then we can add even more info, examples, and guidelines there.

Reply:
Yeah, I'm already working on it actually. Which I should have said of course. I hope we didn't do any double work? 🙂